lib/deps/deps.js

```diff
@@ -59,6 +59,8 @@ module.exports = {
                             mod.vals.forEach(function (val) {
                                 res.push({ block: block.name, mod: mod.name, val: val.name });
                             });
+                        } else {
+                            res.push({ block: block.name, mod: mod.name });
```

Comment:
There is a transformation of the BEMDECL v1.0 to inner ENB format. `bem-naming` should not be aware of declarations and their formats. To solve this task may be suitable `bem-decl`.